### ISSUE TITLE
Bug: Darkmode clicked links are difficult to see 

### ISSUE THREAD

**Issue by I3uckwheat** [TheOdinProject/theodinproject#2587]
### Description

When middle clicking a link on a course page to open the lesson in a new tab, the text turns black and is difficult to read. This was identified in Firefox.

https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript

#### Steps to Reproduce

1. Navigate to: https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript
2. middle mouse click on any Project or link
3. observe the link turning black like the screenshot below
![image](https://user-images.githubusercontent.com/4411701/151726119-256edea8-8321-448b-a45c-fff065cb7e22.png)

#### Expected Behavior

The link should not remain black after middle clicking.

 #### Additional Information

**Comment by leinierperez** [comment]
Would like to tackle this

**Comment by KevinMulhern** [comment]
We're in the middle of a major rewrite of the site's CSS and will be fixing this alongside it.

**Comment by ChargrilledChook** [comment]
Solved by recent Tailwind updates
